app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
```

Comment:
As far as I saw, all the other controller codes also used `console`. So, I stuck with `console` . If this needs to be fixed then I guess it needs to be fixed for all. Then it deserves to be an issue.